tools/gen-release-vote.py

```diff
@@ -77,7 +77,7 @@ def componentList(config, version):
         })
 
 def artifactLinks(name, version, rc):
-    root = 'https://dist.apache.org/repos/dist/dev/openwhisk/apache-openwhisk-%s-%s' % (version, rc)
+    root = 'https://dist.apache.org/repos/dist/dev/openwhisk/%s' % (rc)
```

Comment:
Did we rename things?

Reply:
I changed the move-to-staging script in a previous cleanup and missed updating this to match